tests/PHPStan/Rules/Functions/CallToFunctionParametersRuleTest.php

```diff
@@ -78,6 +78,11 @@ public function testCallToArrayMapVariadic()
 		$this->analyse([__DIR__ . '/data/call-to-array-map-unique.php'], []);
 	}
 
+	public function testCallToArrayUdiff()
+	{
+		$this->analyse([__DIR__ . '/data/call-to-array-udiff.php'], []);
```

Comment:
add some lines with wrong number of arguments to see the available number and the fact that the function is variadic